examples/mnist_dataset_api.py

```diff
@@ -57,7 +55,7 @@ def cnn_layers(inputs):
 
 batch_size = 128
 buffer_size = 10000
-steps_per_epoch = np.ceil(60000 / 128).astype('int')  # = 469
+steps_per_epoch = int(np.ceil(60000 / float(batch_size)))  # = 469
```

Comment:
Why not purely int ops?

Reply:
It would be harder to read (see discussion in unmerged PR https://github.com/fchollet/keras/pull/8284), that kind of speed win is not important in examples, all other examples follow this style.